source/FreeRTOS_DNS_Parser.c

```diff
@@ -450,7 +445,7 @@
                         {
                             /* _HT_ just while testing. When the program gets here,
                              * pucUDPPayloadBuffer was invalid. */
-                            configASSERT( pdFALSE );
+                            FreeRTOS_printf( ( "DNS_ParseDNSReply: pucUDPPayloadBuffer was invalid\n" ) );
```

Comment:
Any reason for removing assert? Is that an IAR warning?

Reply:
The statement below it was causing a - statement unreachable warning.